### PR TITLE
refactor map module

### LINKED ISSUE #735: cleanup olMap package

### PR DESCRIPTION
**Pull request by taulinger**
- pullup olMap package to separate module
- cleanup olMap module

resolves #735

### CODE REVIEW

File: src/modules/toolbox/components/drawToolContent/DrawToolContent.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import css from './drawToolContent.css';
 import { StyleSizeTypes } from '../../../../services/domain/styles';
 import { clearDescription, clearText, finish, remove, reset, setDescription, setStyle, setType } from '../../../../store/draw/draw.action';
 import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
-import { AssetSourceType, getAssetSource, hexToRgb } from '../../../map/components/olMap/olStyleUtils';
+import { AssetSourceType, getAssetSource, hexToRgb } from '../../../olMap/utils/olStyleUtils';
```
Comment (thiloSchlemmer) [inline src/modules/toolbox/components/drawToolContent/DrawToolContent.js:10]: `hexToRgb`, `rgbToHex` etc. are better kept in a separate ./src/utils/colors.js, aren't they?
`AssetSourceType`, `getAssetSource` in  the same way in ./src/utils/asset.js. 

There is no obvious direct relation to the olMap-module, so they can be extracted.

**Review by thiloSchlemmer — APPROVED** [review]
Well done!

File: src/modules/toolbox/components/drawToolContent/DrawToolContent.js (hunk 1 above)
Comment (taulinger) [reply src/modules/toolbox/components/drawToolContent/DrawToolContent.js:10]: Absolutely correct. It violates our encapsulated module rule (see Readme). But let's fix that by a separate PR. Would you open an issue for that?

Comment (thiloSchlemmer) [reply src/modules/toolbox/components/drawToolContent/DrawToolContent.js:10]: Sure

**Review by thiloSchlemmer — APPROVED** [review]
Look good to me.

**Pull request merged**